src/test/scala/io/confluent/examples/streams/ProbabilisticCountingScalaIntegrationTest.scala

```diff
@@ -94,29 +94,19 @@ class ProbabilisticCountingScalaIntegrationTest extends AssertionsForJUnit {
       p
     }
 
-    val builder: StreamsBuilder = new StreamsBuilder()
+    val builder = new StreamsBuilder
     val cmsStoreName = "cms-store"
     builder.addStateStore(createCMSStoreBuilder(cmsStoreName))
 
     // Read the input from Kafka.
-    val textLines: KStream[Array[Byte], String] = builder.stream(inputTopic)
-
-    // Scala-Java interoperability: to convert `scala.collection.Iterable` to  `java.util.Iterable`
-    // in `flatMapValues()` below.
-    import collection.JavaConverters.asJavaIterableConverter
+    val textLines: KStream[Array[Byte], String] = builder.stream[Array[Byte], String](inputTopic)
 
     val approximateWordCounts: KStream[String, Long] = textLines
```

Comment:
Can we get rid of the type declaration here?

Reply:
see above